README.md

```diff
@@ -0,0 +1,318 @@
+# Creating your own UI with the Smooch iOS SDK
+
+**This guide shows you how to create your own UI for the Smooch iOS SDK.**
+
+The Smooch iOS SDK comes with a rich prebuilt user interface with the option to configure the interface using the Smooch dashboard or the REST API.
+
+If needed you can completely replace Smooch's default UI with your own interface.
+
+Although you can replace the default UI, note that this means rewriting support for all the Smooch message types, that you want to support in your custom UI.
+
+## Overview
+
+The iOS SDK can be initialized without displaying it's default UI. You can then make use of the SDK's messaging APIs to send messages, and it's conversation delegate methods to receive messages.
+
+This guide is separated into two parts:
+- [Part One](#) for setting up a generic messaging UI in iOS
+- [Part Two](#) for dropping Smooch methods in to add the messaging functionality.
+
+It will help to have the [SDK documentation](https://docs.smooch.io/api/ios/index.html) on hand while following this guide.
+
+The complete code for this guide is included in this repository.
+
+## Part One: setting up the UI
+
+In part one we'll set up the UI that we can drop Smooch into in part two.
```

Comment:
"In part one we'll set up the UI that we can drop Smooch into in part two." -> "In part one we'll set up the UI that we'll be integration Smooch with in part two."